server/static/css/oauth.css

```diff
@@ -1,15 +1,40 @@
-.oauth {
-    text-align: left;
-    padding-top: 0;
-    padding: 0;
+.container {
```

Comment:
My mistake: OAuth and landing are mixed together in assets - so we can't have two definitions here. Fixing now.